packages/mobile/android/app/src/main/AndroidManifest.xml

```diff
@@ -59,6 +60,7 @@
                 <data android:scheme="http" />
                 <data android:host="redirect.audius.co" />
                 <data android:host="audius.co" />
+                <data android:pathPattern="/oauth/auth.*" />
```

Comment:
this is the only new change, everything else is formatting

Reply:
this change is backwards. we are also reverting this on RC because current android app instantly crashes